targeted/json_csrf_poc_generator

```diff
@@ -1,38 +1,43 @@
 //csrf poc generater supporting json csrf
-//it will print results to the zap script console. you will have to copy paste it from there
+//it will copy the results to clipboard and print them to the zap script console
 // released under the Apache v2.0 licence.
 //You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
 function invokeWith(msg) {
-	print("\n\n\n");
-	print("<!DOCTYPE html>\n");
-	print("<head>\n <title>CSRF POC</title>\n</head>");
-	print("<body>");
+	string = "<!DOCTYPE html>\n";
+	string += "<head>\n <title>CSRF POC</title>\n</head>";
+	string += "\n<body>";
 	if(msg.getRequestHeader().getMethod().toString()== "POST"){
 	body = msg.getRequestBody().toString();
 	body = body.trim();
 	if(isJson(body))
-		print("<form action=\""+ msg.getRequestHeader().getURI().toString() +"\" id=\"formid\" method=\"post\" enctype=\"text/plain\">" );
+		string += "\n<form action=\""+ msg.getRequestHeader().getURI().toString() +"\" id=\"formid\" method=\"post\" enctype=\"text/plain\">" ;
 	else
-		print("<form action=\""+ msg.getRequestHeader().getURI().toString() +"\" id=\"formid\" method=\"post\">");
+		string += "\n<form action=\""+ msg.getRequestHeader().getURI().toString() +"\" id=\"formid\" method=\"post\">";
 	if(!isJson(body)){	
 	body=body.split('&');
 	for(i=0;i<body.length;i++)
 	{
 		keyval = body[i].split('=')
-		print("<input type=\"hidden\" name=\""+decodeURIComponent(keyval[0])+"\" value=\""+decodeURIComponent(keyval[1])+"\" />");	
+		string += "\n<input type=\"hidden\" name=\""+decodeURIComponent(keyval[0])+"\" value=\""+decodeURIComponent(keyval[1])+"\" />";	
```

Comment:
The name/value of the parameter should be HTML escaped (unless that's on purpose?).